universal-login-react/src/core/utils/formatCrypto.ts

```diff
@@ -0,0 +1,3 @@
+export const formatCrypto = (crypto: string, places: number) => {
+  return parseFloat(crypto).toFixed(places).toString();
```

Comment:
crypto: utils.BigNumberish
return utils.formatUnits(utils.bigNumberify(value.toString()), number)

Reply:
Add test for it!!!!